hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/queries/contract/HapiGetContractInfo.java

```diff
@@ -141,10 +141,8 @@ protected void assertExpectationsGiven(HapiSpec spec) throws Throwable {
                 || !absentRelationships.isEmpty()
                 || expectations.isPresent()
                 || registryEntry.isPresent()) {
-            final var detailsLookup = QueryVerbs.getAccountDetails(
-                            spec.setup().defaultShard().getShardNum() + "."
-                                    + spec.setup().defaultRealm().getRealmNum() + "."
-                                    + actualInfo.getContractID().getContractNum())
+            final var detailsLookup = QueryVerbs.getAccountDetails(spec.shard() + "." + spec.realm() + "."
```

Comment:
Not a blocker, but is `getContractID` not sufficient?

```suggestion
            final var detailsLookup = QueryVerbs.getAccountDetails(actualInfo.getContractID());
```

Reply:
Yes, It is fixed in this [PR](https://github.com/hiero-ledger/hiero-consensus-node/pull/19077)